modules/rubiconBidAdapter.js

```diff
@@ -481,22 +501,30 @@ export const spec = {
     }
 
     // visitor properties
-    if (params.visitor !== null && typeof params.visitor === 'object') {
-      Object.keys(params.visitor).forEach((key) => {
```

Comment:
We need to merge the params.visitor and fpd.user values, and also merge params.inventory and fpd.site values.

Reply:
This is being done on lines 504 and 514 below.
`config` will take precedence over `params`, though if both have a matching property name.